### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Properties_BlobProperties_ScreenReader: Voiceover does not read the label of the edit boxes when focus goes to them.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#1979]
“Check out Accessibility Insights! Identify accessibility bugs before check-in and make bug fixing faster and easier.”
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;#MAS2.4.2 #VoiceOver; #Mac; 
Environment Details:
****Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)
**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Right click on blob and select "Properties" control and hit enter.
7. Start VoiceOver and navigate using Tab key and check whether the Voiceover reads the label of the edit boxes when focus goes to them.
**​Actual:​**
Voiceover does not read the label of the edit boxes when focus goes to them on the properties screen.
**Expected:**
VoiceOver should read the label of the edit boxes when focus goes to them on the properties screen.
**UserImpact:**
Screen reader users unable to know the which dialog opened if it will not read the title.
Recommendation:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference:**
[1979_A11y_AzureToolsStorageExplorer_Properties_BlobProperties_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3669732/1979_A11y_AzureToolsStorageExplorer_Properties_BlobProperties_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
It's looking like this is a chromium issue since it is:

- platform specific
- screen reader specific
- browser specific
- occurs outside of electron

I've opened a but in chromium to get help from them on it: https://bugs.chromium.org/p/chromium/issues/detail?id=1024011

**Comment by Suraj1200499** [comment]
@MRayermannMSFT 
Since the primary bug#1024011 is in open State. We will take action on the bug#1979 Once the Primary bug is Closed.

**Comment by MRayermannMSFT** [comment]
Make sure to add to VPAT

**Comment by Alexandria-R** [comment]
Approved for exclusion as low impact.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
